### ISSUE TITLE
Add Loss Functions to Ivy Frontend

### ISSUE THREAD

**Issue by djl11** [unifyai/ivy#48]
Add [Loss Functions](https://github.com/unifyai/ivy/blob/master/ivy/neural_net_functional/losses.py) to Ivy frontend:

- [ ] cross_entropy
- [ ] binary_cross_entropy
- [ ] sparse_cross_entropy
